src/bulma/pages/components/Order.vue

```diff
@@ -173,7 +173,7 @@ export default {
         },
         fulfilling() {
             return this.form
-             && this.form.field(this.form.param('fulfillingAttribute')).value !== null;
```

Comment:
This change is needed because when the not-fulfilled order  loads, the form field value will be `null`, thus the pseudo computed value will be correct.

When loading a fulfilled order, and then unfulfilling it, the form field value will be an empty string (see above why) and the computed value will also be correct.